modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testFunctional/macros/ExportTask.macro

```diff
@@ -11,6 +11,30 @@ definition {
 			expected = ${expectedCount});
 	}
 
+	@summary = "Default summary"
+	macro createExportTask(className = null, contentType = null) {
+		Variables.assertDefined(parameterList = "${className},${contentType}");
+
+		if (contains(${className}, "Organization")) {
+			var url = "export-task/${className}/${contentType}?flatten=true";
```

Comment:
do you know if flatten=true is needed for every export-task? Maybe we can set it by default for now since  there is no other test reference to your new created createExportTask macro.(I mean we can remove the else block for now)